src/workers/cdc/common.ts

```diff
@@ -8,3 +19,76 @@ export const getTableName = <Entity>(
   con: DataSource,
   target: EntityTarget<Entity>,
 ): string => con.getRepository(target).metadata.tableName;
+
+export const notifyPostContentUpdated = async ({
+  con,
+  logger,
+  post,
+}: {
```

Comment:
Not related to this PR

We could start by not passing the logger down in everything, if we need the logger, we can just import it where we actually need it. Less hoisting.
What do you think @capJavert?

Reply:
yeah, i forgot we had it, I'll replace

Reply:
422e119738d28d1a7f65558e2f5c13585fac17d0